lib/plugins/aws/rollback/index.test.js

```diff
@@ -20,12 +21,13 @@ describe('AwsRollback', () => {
       region: 'us-east-1',
       timestamp: 1476779096930,
     };
-    serverless.setProvider('aws', new AwsProvider(serverless));
+    provider = new AwsProvider(serverless);
```

Comment:
@erikerikson The AwsProvider constructor in the tests should include the options parameter, as AwsProvider.options is evaluated to fetch the stage in `getStage()`. Otherwise it will default to whatever is set as default or set in `serverless.config.stage` or `serverless.service.provider.stage`.
E.g. the set options here set it to dev (which is the default), but when setting it here to `someStage` it will fail, because you did not give the actual options to the constructor.

Reply:
Looks like there were others (a search for `new AwsProvider(serverless)` uncovered them easily).  Given that no tests are failing, I'll add them in the cases where an options hive is defined but will otherwise not provide the parameter.  Let me know if you think I should be more aggressive on that front.

Reply:
This and others are addressed in the latest commit.